README.md

```diff
@@ -94,15 +116,25 @@ Your Answer:
 What are the differences between calling and referencing a function? Please provide examples of each.
 
 ```text
+For function();
 
+calling would be inputting the foillowing: function();
+
+referencing would be simply inputting "function;"
```

Comment:
:+1: , but you'd never actually call a function `function` -- that is, you'd never write `function function(){}`. Javascript will throw an error at you.